opentelekomcloud/acceptance/dcs/resource_opentelekomcloud_dcs_instance_v2_test.go

```diff
@@ -0,0 +1,611 @@
+package acceptance
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+	"github.com/opentelekomcloud/gophertelekomcloud/openstack/dcs/v2/instance"
+	"github.com/opentelekomcloud/terraform-provider-opentelekomcloud/opentelekomcloud/acceptance/common"
+	"github.com/opentelekomcloud/terraform-provider-opentelekomcloud/opentelekomcloud/acceptance/env"
+	"github.com/opentelekomcloud/terraform-provider-opentelekomcloud/opentelekomcloud/common/cfg"
+)
+
+const dcsV2InstanceName = "opentelekomcloud_dcs_instance_v2.instance_1"
+
+func TestAccDcsInstancesV2_basic(t *testing.T) {
+	var dcsInstance instance.DcsInstance
+	var instanceName = fmt.Sprintf("dcs_instance_%s", acctest.RandString(5))
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:          func() { common.TestAccPreCheck(t) },
+		ProviderFactories: common.TestAccProviderFactories,
+		CheckDestroy:      testAccCheckDcsV2InstanceDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccDcsV2InstanceBasic(instanceName),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckDcsV2InstanceExists(dcsV2InstanceName, dcsInstance),
```

Comment:
could you use here resource_check from common?

Reply:
done